### PR TITLE
Spectrum tokens 17 for Action Bar

### LINKED ISSUE #807: Check actionbar against DL and VRT
Check [actionbar](https://github.com/adobe/spectrum-css/tree/spectrum-tokens-17/components/actionbar)
- [x] matches DL
- [x] check VRT passes
- [x] check and update migration guides [example](https://github.com/adobe/spectrum-css/blob/f58d8e071e5cce8db653e61c5a09b7690c236f63/components/pagination/metadata/pagination-explicit.yml#L3)

### PR DESCRIPTION
**Pull request by jianliao**
## Description
Fixed #807 and when parent branch was merged to main, it will fix `#894`

1. Example change
- Changed string capitalization
- Switched to use `ActionGroup`
- Changed Workflow icon size from small to medium

2. Add `@spectrum-css/actiongroup` as dependency

3. Change action bar content padding-right to be half of the padding-left to match [UI kit](https://spectrum-contributions.corp.adobe.com/page/action-bar-beta/)

## How and where has this been tested?
Chrome/Firefox/Safari on MacOS

## Screenshots
1. Desktop scale
image
2. Mobile scale
image

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: components/actionbar/metadata/actionbar.yml [hunk 1]
```diff
@@ -17,16 +17,16 @@ examples:
                 <use xlink:href="#spectrum-css-icon-DashSmall" />
               </svg>
             </span>
-            <span class="spectrum-Checkbox-label">2 Selected</span>
+            <span class="spectrum-Checkbox-label">2 selected</span>
           </label>
-          <div class="spectrum-ButtonGroup">
-            <button class="spectrum-ActionButton spectrum-ActionButton--quiet">
+          <div class="spectrum-ActionGroup">
+            <button class="spectrum-ActionButton spectrum-ActionButton--quiet spectrum-ActionGroup-item">
```
Comment (GarthDB) [inline components/actionbar/metadata/actionbar.yml:23]: @jianliao could you add a migration guide section and include the need to use `spectrum-ActionGroup-item` and maybe mention `spectrum-Icon--sizeM` as well

**Review by GarthDB — CHANGES_REQUESTED** [review]

File: components/actionbar/metadata/actionbar.yml (hunk 1 above)
Comment (jianliao) [reply components/actionbar/metadata/actionbar.yml:31]: Done.

**Review by lazd — APPROVED** [review]
Super quick review, but hooray for ActionGroup and sizeM icons!

File: components/actionbar/index.css [hunk 2]
```diff
@@ -13,7 +13,8 @@ governing permissions and limitations under the License.
 
 :root {
   --spectrum-actionbar-height: var(--spectrum-global-dimension-size-600);
-  --spectrum-actionbar-padding-x: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-padding-left: var(--spectrum-global-dimension-size-200);
+  --spectrum-actionbar-padding-right: calc(var(--spectrum-global-dimension-size-200) / 2);
```
Comment (bernhard-adobe) [inline components/actionbar/index.css:17]: hmm, not sure if you'd like to use the DNA-vars for that instead of the global dimensions?

**Review by bernhard-adobe — APPROVED** [review]
minor question about DNA-vars. Maybe Spectrum-CSS likes to use the global vars more than Spectrum SVG-Generator

**Pull request merged**